code/game/turfs/simulated/floor/plating.dm

```diff
@@ -24,18 +25,26 @@
 		to_chat(user, "<span class='notice'>You might be able to build ontop of it with some <i>tiles</i>...</span>")
 
 /turf/open/floor/plating/Initialize()
+
+	if(icon == 'icons/turf/floors/plating.dmi') //Hack, lots of subclasses of plating
```

Comment:
This looks reall awful and hardcoded. It is, but having a fuzzy check for any smoothing-enabled *.dmi would be a more expensive operation.

Reply:
Why not just assign a variable to all turfs that use the icon? I'm confused by this.

Reply:
Because that means setting a snowflake var on all subclasses.